docs/src/modules/components/MarkdownDocs.js

```diff
@@ -256,8 +250,8 @@ function MarkdownDocs(props) {
                     href={prevPage.pathname}
                     size="large"
                     className={classes.pageLinkButton}
+                    startIcon={<ChevronLeftIcon fontSize="small" />}
```

Comment:
```suggestion
                    startIcon={<ChevronLeftIcon />}
```

Reply:
![Capture d’écran 2019-10-01 à 19 53 22](https://user-images.githubusercontent.com/3165635/65986907-23910280-e485-11e9-99b4-6b28a78e6ccf.png)

I don't have the permission to push 👼